_pages/events/code-of-conduct.md

```diff
@@ -0,0 +1,65 @@
+
+## Summary
+
+We are dedicated to providing a harassment-free experience for everyone, regardless of background or identity. We do not tolerate harassment of event participants in any form.
+
+**Any form or behaviour to exclude, intimidate, or cause discomfort is a violation of the Code of Conduct.**
+
+All communication should be appropriate for a professional audience including people of many different backgrounds.
+
+Be kind to others. Do not insult or put down other attendees.
+
+Behave professionally. Remember that harassment and exclusionary jokes are not appropriate at our events.
+
+Attendees violating these rules may be asked to leave the event at the sole discretion of the event organisers.
+
+In order to foster a positive and professional environment we encourage the following behaviours:
+
+- Use welcoming and inclusive language
+- Be respectful of different viewpoints and experiences
+- Gracefully accept constructive criticism
+- Focus on what is best for the community
+- Show courtesy and respect towards other community members
+
+## Clarifications
+
+Harassment includes offensive communication related to gender, sexual orientation, disability, physical appearance, body size, race, religion, sexual images in public spaces, deliberate intimidation, stalking, following, harassing photography or recording, sustained disruption of talks or other events, inappropriate physical contact, and unwelcome sexual attention.
+
+Any form or behaviour to exclude, intimidate, or cause discomfort is a violation of the Code of Conduct.
+
+Participants asked to stop any harassing behaviour are expected to comply immediately.
+
+Be careful in the words that you choose. Remember that words can be offensive to those around you. Swearing and offensive jokes are not appropriate at our events.
+
+If a participant engages in behaviour that violates this Code of Conduct, the event organisers may take any action they deem appropriate, including warning the offender or expulsion from the event.
+
+## Contact information
+
+At all our events, we have staff dedicated to enforcing the Code of Conduct and dealing with its violations. 
+
+If you are being harassed, notice that someone else is being harassed, or have any other concerns, please contact one of the Code of Conduct staff responsible for the event you are attending and they will be happy to assist those experiencing harassment to feel safe for the duration of the event.
+
+- Jeremy Cohen (Imperial College London) - [jeremy.cohen@imperial.ac.uk](mailto:jeremy.cohen@imperial.ac.uk)
+- Eirini Zormpa (Imperial College London) - [e.zormpa@imperial.ac.uk](mailto:ezormpa@imperial.ac.uk)
+- Neil Chue Hong (University of Edinburgh) - [n.chuehong@epcc.ed.ac.uk](mailto:n.chuehong@epcc.ed.ac.uk)
+
+## Procedure for reporting and responding to Harassment
+- [Attendee procedure for incident reporting](http://www.universe-hpc.ac.uk/events/code-of-conduct/report-incident)
```

Comment:
We should use Jekyll relative links to ensure this works locally as well as when deployed (and it's then also then indepedent of wether HTTP/HTTPS is used:

```suggestion
- [Attendee procedure for incident reporting]({{ site.baseurl}}/events/code-of-conduct/report-incident)
```